Exercise_5.java

```diff
@@ -1,20 +1,69 @@
 class IterativeQuickSort { 
     void swap(int arr[], int i, int j) 
     { 
-	//Try swapping without extra variable 
+	//Try swapping without extra variable
+    	int t = arr[i]; 
+        arr[i] = arr[j]; 
+        arr[j] = t; 

```

Comment:
here you are using extra space i.e. try it without using extra space!!

Reply:
Made necessary corrections - Please Review.